### ISSUE TITLE
: End-to-end numpyro-based sampling of cosmology and initial conditions on LSST footprint

### ISSUE THREAD

**Issue by EiffL** [LSSTDESC/bayesian-pipelines-cosmology#13]
Here is a detailed description of what we want to do:

 - Technical point: Gaussian shells, jitted jax, numpyro-based
 - realistic LSST photo-z distribution (fixed)
 - realistic LSST noise and footprint (LSST Y10 - Y1)
 - data generated from the forward model
 - sampling in two steps:
     - fixed latent variables, sampling cosmology
     - joint sampling of cosmology and latent variables
- nside 1024,  l ~ O(2000)
- simulated shear + IA

**Comment by EiffL** [comment]
Technical things that need to be solved: 

- [ ] Building a baseline model (@AZhou00 will open a PR)
- [ ] Slow jitting of the forward (maybe we need to jit the SHT)
- [ ] Adding survey realism (@nataliaporqueres  will prototype a forward model with footprint+noise, etc.)
- [ ] Shear normalization issue

**Comment by nataliaporqueres** [comment]
I pushed the function for the computing shear from convergence with the normalisation issue. The example script is `testjax.py`
